micro-benchmarks/matmul_relu.py

```diff
@@ -0,0 +1,84 @@
+import sre_compile
+import torch
+from benchmark_helper import time_with_torch_timer
+from torchinductor.triton_ops.matmul import matmul_out as triton_mm_out
+from torchinductor.triton_ops.matmul import matmul as triton_mm
+import torchdynamo
+import torchinductor.config as config
+
+config.triton.use_mm = True
+
+@torchdynamo.optimize("inductor", nopython=True)
+def inductor_mm(a, b):
+    return torch.mm(a, b)
+
+def torch_mm_relu(a, b):
+    return torch.nn.functional.relu(torch.matmul(a, b))
```

Comment:
Nit: If you're just doing this for convenience of not having to write out the whole thing, this might be more pythonic:

```
mm = torch.matmul
torch_relu = torch.nn.functional.relu
...
torch_relu(mm(a,b))
```